RNSnackbar.podspec

```diff
@@ -0,0 +1,20 @@
+require 'json'
+
+package = JSON.parse(File.read('package.json'))
+
+Pod::Spec.new do |s|
+  s.name                = 'RNSnackbar'
+  s.version             = package['version']
+  s.summary             = package['description']
+  s.description         = package['description']
+  s.homepage            = package['homepage']
+  s.license             = package['license']
+  s.author              = package['author']
+  s.source              = { :git => 'https://github.com/cooperka/react-native-snackbar.git' }
```

Comment:
__dir__ is a function and returns absolute path. But in this case, .podspec and package.json are in a same directory level, so omit it is also fine.